openpower/configs/hostboot/witherspoon.config

```diff
@@ -52,15 +52,15 @@ unset PNOR_TWO_SIDE_SUPPORT
 
 unset BMC_BT_LPC_IPMI
 
-# Enable Checktop Analysis
+# Disable Checktop Analysis
```

Comment:
Please put in a separate patch, this has nothing to do with MEMVPD configuration, it's correcting a comment.